lib/admin/WP_Auth0_Admin.php

```diff
@@ -62,57 +62,41 @@ public function __construct( WP_Auth0_Options $a0_options, WP_Auth0_Routes $rout
 
 	public function init() {
 		add_action( 'admin_init', array( $this, 'init_admin' ) );
-		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue' ), 1 );
 	}
 
 	public function admin_enqueue() {
+
 		if ( ! isset( $_REQUEST['page'] ) || 'wpa0' !== $_REQUEST['page'] ) {
 			return;
 		}
 
-		$client_id = $this->a0_options->get( 'client_id' );
-		$secret = $this->a0_options->get( 'client_secret' );
-		$domain = $this->a0_options->get( 'domain' );
-
-		if ( empty( $client_id ) || empty( $secret ) || empty( $domain ) ) {
-			add_action( 'admin_notices', array( $this, 'create_account_message' ) );
-		}
-
-		$this->validate_required_api_scopes();
-
 		wp_enqueue_media();
-		wp_enqueue_script( 'wpa0_admin', WPA0_PLUGIN_URL . 'assets/js/admin.js' );
-		wp_enqueue_script( 'wpa0_async', WPA0_PLUGIN_URL . 'assets/lib/async.min.js' );
-		wp_enqueue_style( 'wpa0_bootstrap', WPA0_PLUGIN_URL . 'assets/bootstrap/css/bootstrap.min.css' );
-		wp_enqueue_script( 'wpa0_bootstrap', WPA0_PLUGIN_URL . 'assets/bootstrap/js/bootstrap.min.js' );
-		wp_enqueue_style( 'wpa0_admin_initial_settup', WPA0_PLUGIN_URL . 'assets/css/initial-setup.css' );
+		wp_enqueue_script(
```

Comment:
Adding version numbers to act as cache breakers when needed. Otherwise the WP version is used.